tests/launcher.py

```diff
@@ -479,6 +488,58 @@ def make_mesh(self):
 
     def make_custom(self):
         print('making custom')
+
+    def launch(self, instance: nodeDefinition):
+        dd = Path(instance.data_dir_name)
+        out = dd / 'stdout.txt'
+        err_sl = dd / 'stderr.txt'
+        err = dd / Path(f'stderr.{self.launch_time}.txt')
+        pidf = dd / Path(f'{Utils.EosServerName}.pid')
+
+        eosdcmd = [Utils.EosServerPath]
+        if self.args.skip_signature:
+            eosdcmd.append('--skip-transaction-signatures')
+        if getattr(self.args, Utils.EosServerName):
+            eosdcmd.extend(shlex.split(getattr(self.args, Utils.EosServerName)))
+        if instance.index in self.args.specific_nums:
+            i = self.args.specific_nums.index(instance.index)
+            eosdcmd.extend(shlex.split(getattr(self.args, f'specific_{Utils.EosServerName}es')[i]))
+        eosdcmd.append('--config-dir')
+        eosdcmd.append(str(instance.config_dir_name))
+        eosdcmd.append('--data-dir')
+        eosdcmd.append(str(instance.data_dir_name))
+        eosdcmd.append('--genesis-json')
+        eosdcmd.append(f'{instance.config_dir_name}/genesis.json')
+        if self.args.timestamp:
+            eosdcmd.append('--genesis-timestamp')
+            eosdcmd.append(self.args.timestamp)
+
+        if 'eosio::history_api_plugin' in eosdcmd and 'eosio::trace_api_plugin' in eosdcmd:
```

Comment:
Is this for backward compatibility, or why are we referencing the history plugin?

Reply:
Yes, that's backwards compatibility code preserved from `main`.